dolphinscheduler-dist/release-docs/licenses/ui-licenses/LICENSE-@form-create-element-ui

```diff
@@ -1,21 +0,0 @@
-MIT License
-
```

Comment:
This file should be kept. `git checkout 47a751f~1 -- dolphinscheduler-dist/release-docs/licenses/ui-licenses/LICENSE-@form-create-element-ui`

Reply:
Thanks for your code  review，And I want to know whether this code contribution is successful ?